src/components/articles/ArticleCategory.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="mb-5 ml-5 mr-5 flex justify-between phone:ml-8">
+    <F1Icon :class="getCategoryClass('F1')" @click="updateCategory('F1')" />
```

Comment:
I would create an array of object, with the icons and would v-for on a <component>